### DISCUSSION TITLE
.fyrox An example of an open standard ROM format

### DISCUSSION THREAD

**Discussion by ghost** [FyroxEngine/Fyrox#681] [Ideas]
`.fyrox`
- Privacy. 
> .apk, .exe, etc. Most of their games are stealing user privacy and mining. etc.
- Security.
`.fyrox` requires an emulator. Or a device that supports the .fyrox format.
> Application Isolation
Independent Application Environments: Each application runs in its own sandbox, limiting its access to system resources and data from other applications. This isolation reduces the potential threat of malicious applications accessing user data.
User Data Protection
Data Separation: Data between different applications and user environments is isolated, ensuring that data in one environment cannot be accessed by another. This is crucial for protecting sensitive information.
Memory Tagging:

- `.fyrox` Unified File Format 
Used to share and store game resources between different consoles or platforms. The purpose of this format is to simplify game development and cross-platform compatibility, making it easier for developers to publish and run games across devices.
> DS/PSP/PS2/PS3/PS4/Wii/XBOX 360, etc. use a new file extension with each new device they release.

This creates a huge problem for game development, maintenance and distribution.

**Comment by mrDIMAS** [comment]
I'm sorry, but what's this gibberish about? I re-read it multiple times and still don't understand.

**Reply by ghost** [reply]
> I'm sorry, but what's this gibberish about? I re-read it multiple times and still don't understand.

My idea is to be able to encapsulate game projects as .fyrox instead of .exe .apk, etc.
You can think of it as Nintendo's .XCI.
